### PR TITLE
feat: Support adding graphics devices: VZVirtioGraphicsDeviceConfiguration and VZMacGraphicsDisplayConfiguration

### PR DESCRIPTION
**Pull request by siketyan**
https://developer.apple.com/documentation/virtualization/vzmacgraphicsdisplayconfiguration?language=objc
https://developer.apple.com/documentation/virtualization/vzvirtiographicsdeviceconfiguration?language=objc

### CODE REVIEW

File: src/virtualization/virtual_machine.rs [hunk 1]
```diff
@@ -276,4 +300,8 @@ impl VZVirtualMachine {
             _ => VZVirtualMachineState::Other,
         }
     }
+
+    pub unsafe fn id(&self) -> Id {
+        *self.0
+    }
```
Comment (siketyan) [inline src/virtualization/virtual_machine.rs:338]: Added this for sending [VZVirtualMachineView setVirtualMachine:] out of this crate. I think the view is not in the scope of this crate.

**Comment by suzusuzu** [comment]
@siketyan 
I would appreciate it if you could resolve the conflict.

**Comment by siketyan** [comment]
@suzusuzu I have just resolved the conflict.

**Comment by suzusuzu** [comment]
Thank you for your quick response. I have merged it.

**Pull request merged**